### ISSUE TITLE
gstreamer missing elements when built for NWJS

### ISSUE THREAD

**Issue by Zuzon** [dturing/node-gstreamer-superficial#61]
When addon is used with pure nodejs, all works totally fine, but when it is compiled for nwjs, then i'm receiving errors of missing gstreamer elements, like avdec_h264. `GstError: no element "avdec_h264"`
It looks like under nwjs environment gstreamer-superficial is not using a gst, which is already installed on system, but it tries to use another kind.

Later i have added ffmpeg to my nwjs and received different error:
`Crashing due to FD ownership violation`

nwjs 0.69.1
ubuntu 22

**Comment by dfischer23** [comment]
I have no idea about the "FD ownership violation", but for the first issue, have you tried setting/checking GST_PLUGINS_PATH for your nwjs environment?

**Comment by Zuzon** [comment]
Yes, GST_PLUGIN_PATH and GST_PLUGIN_SYSTEM_PATH tried. it is being ignored. Also tried to pass wrong plugin path in terminal. with terminal works as expected. but with nwjs looks like it is being ignored

**Comment by Zuzon** [comment]
As i understood, this native model is automatically linking with ffmpeg which is coming with nwjs, not system one.
I have enabled gstreamer debug messages and seen all paths are correct, but then initialization fails. any idea how to link gstreamer with system installed ffmpeg?
